### DISCUSSION TITLE
How to change led indicator from gpio 2 to gpio 22

### DISCUSSION THREAD

**Discussion by sfttech2000** [BitMaker-hub/NerdMiner_v2#376] [Q&A]
I edit file src/drivers/devices/esp32DevKit.h
#ifndef LED_PIN
#define LED_PIN 22
#endif
but not work led light is still on, not blink. how to fix it? (i cannot upload picture) thank you.

**Comment by jsy-rg1570** [comment]
> I edit file src/drivers/devices/esp32DevKit.h #ifndef LED_PIN #define LED_PIN 22 #endif but not work led light is still on, not blink. how to fix it? (i cannot upload picture) thank you.

Dear, what board is it ?

Better define LED_PIN in the platformio.ini file for the board you are building.

build_flags = 
...
    -D LED_PIN=22
...

**Reply by sfttech2000** [reply]
now i set -D LED_PIN=22 but not work(led light still on not blink) my board name wemos lite v.1.0.0 
![1b722b73-20ae-4e79-a410-2713ca4fe891](https://github.com/BitMaker-hub/NerdMiner_v2/assets/157212806/dc25319b-a948-44e9-bbda-d07bb1a0098e)
![Capture](https://github.com/BitMaker-hub/NerdMiner_v2/assets/157212806/1ee3184d-f97f-4809-99f1-9620942732ff)

**Comment by jsy-rg1570** [comment]
Dear,
Your board has a PWM LED which need different code (check ledcAttachPin, ledcWrite, ledcSetup).
